_source/_includes/setup/oidcdebugger.md

```diff
@@ -0,0 +1,5 @@
+An easy way to get an access token is to generate one using [OpenID Connect Debugger](https://oidcdebugger.com/). First, you must configure your application on Okta to use OpenID Connect's implicit flow.
+
+Run `okta login` and open the resulting URL in your browser. Go to the **Applications** section and select the application you just created. Edit its General Settings and add **Implicit (Hybrid)** as an allowed grant type, with access token enabled. Then, add `https://oidcdebugger.com/debug` to the **Login redirect URIs**. Click **Save** and copy the client ID for the next step.
```

Comment:
This is clever :)
Maybe we should add an `okta info` option 🤷

Reply:
I like `okta token` (or some command to create an access token) because it means you don't have to enable implicit flow for OIDC Debugger. Or maybe we should contract @nbarbettini to update OIDC Debugger?